app/controllers/clock_controller.rb

```diff
@@ -11,13 +10,13 @@ def index
   end
 
   def timezones
-    @timezones = user_signed_in? ? WeatherSetter.new(all_timezones[1..5]).call : all_timezones
+    @timezones = user_signed_in? ? WeatherSetter.new(all_timezones[0..5]).call : all_timezones
   end
 
-  def show
-    @timezone = get_new_support_timezone(params[:id])
-    @weather = next_forecast_days(@timezone.name)
-  end
+  # def show
+  #   @timezone = get_new_support_timezone(params[:id])
+  #   @weather = next_forecast_days(@timezone.name)
+  # end
```

Comment:
Let's remove this method if it's no longer need it.

Reply:
Thanks, I solve that